src/components/emailSelector/Label.tsx

```diff
@@ -132,6 +133,7 @@ const Label = ({
 
         case COMMA:
         case SEMI:
+        case SPACE:
```

Comment:
The reason we didn't do this on space is because you can also type a customer name (first and last name or if the company name has spaces) and then you can select the email from the autocomplete

Could you double check with design it is ok to remove this or perhaps add an option for this functionality?

Reply:
Good point, discussing it with design